gtk3-macros/src/attribute_parser.rs

```diff
@@ -0,0 +1,180 @@
+use proc_macro2::Span;
```

Comment:
No header in this crate?

Reply:
That's how it apparently always was. So we need to fix that after this PR, and also in gtk4 (CC @bilelmoussaoui )

Reply:
Ok, I'll let you open the issue then. ;)

Reply:
Ah you said I should, ok!